.travis.yml

```diff
@@ -1,22 +1,26 @@
+sudo: false
+env:
+  global:
+    - CC_TEST_REPORTER_ID=02a53f73b72a95af756b1c0270d4b22a6f07d25a1c65048e341371e7b72671d9
 language: ruby
 rvm:
   - 2.2
   - 2.3
-sudo: required
-dist: trusty
-before_install: 
-  - gem install bundler
-  - gem update --system
-  - gem --version
 gemfile:
   - gemfiles/3.2.gemfile
   - gemfiles/4.2.gemfile
   - gemfiles/5.0.gemfile
   - gemfiles/5.1.gemfile
   - gemfiles/5.2.gemfile
-addons:
-    code_climate:
-        repo_token: 02a53f73b72a95af756b1c0270d4b22a6f07d25a1c65048e341371e7b72671d9
-after_success:
-  - bundle exec codeclimate-test-reporter
+before_install:
+  - gem install bundler
+  - gem update --system
+  - gem --version
+  - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
```

Comment:
Is there a difference between putting this line in `before_install` section and in `before_script`  section?
Since the [official guide](https://docs.codeclimate.com/docs/travis-ci-test-coverage) of `codeclimate` put it in `before_script`  section.

![image](https://user-images.githubusercontent.com/4011729/40315772-e2c73814-5d4e-11e8-91ed-0ee6c6916709.png)

Reply:
Looks the same, they have so many examples lol
https://github.com/codeclimate/test-reporter/blob/master/examples/ruby_examples.md

Reply:
haha, it seems either way is ok.